FinanceManager.Core/Entities/Accounts/BankAccount.cs

```diff
@@ -62,5 +79,13 @@ public virtual void GetEntry(DateTime start)
         {
             throw new NotImplementedException();
         }
+        public int GetNextFreeId()
```

Comment:
dlaczego robisz to w taki sposób ?

Reply:
Potrzebowałem szybkiego sposobu na znalezienie najbliższego wolnego ID. Być może dobrym pomysłem było by przenieść tą metodę do klasy bazowej

Reply:
ale po co ?